commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -47,10 +47,28 @@
                 <div class="u-responsive-ratio u-responsive-ratio--hd">
                     @page.video.youtubeId.map { youtubeId =>
                         <iframe id="hosted-video-@youtubeId"
-                        class="js-hosted-youtube-video"
+                        data-duration="@{page.video.duration}"
+                        class="js-hosted-youtube-video hosted__youtube-video"
                         src="https://www.youtube.com/embed/@youtubeId?modestbranding=1&showinfo=0&rel=0&enablejsapi=1@{if(host != "") s"&origin=$host" else ""}"
                         frameborder="0"
                         allowfullscreen=""></iframe>
+                        <div class="vjs-control-bar youtube-player">
+                            <div class="vjs-current-time vjs-time-control vjs-control" aria-hidden="true">
+                                <div class="vjs-current-time-display" aria-live="off">
+                                    <span class="vjs-control-text">Current Time </span><span class="js-youtube-current-time">0:00</span>
+                                </div>
+                            </div>
+                            <div class="vjs-time-control vjs-time-divider" aria-hidden="true">
+                                <div>
+                                    <span>/</span>
+                                </div>
+                            </div>
+                            <div class="vjs-duration vjs-time-control vjs-control" aria-hidden="true">
+                                <div class="vjs-duration-display" aria-live="off">
+                                    <span class="vjs-control-text">Duration Time</span>@{"%2d:%02d".format(page.video.duration/60, page.video.duration % 60)}
```

Comment:
@akash1810 I just created a PR with that https://github.com/guardian/frontend/pull/14533